### PR TITLE
chore: prepare version 1.16.1-SNAPSHOT

### PR DESCRIPTION
**Pull request by alacambre-yseop**

### CODE REVIEW

File: client/jest-runner-mocha.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const extensionDevelopmentPath = path.resolve(__dirname, '../../');
+const resourcesDirPath = path.resolve(extensionDevelopmentPath, 'client/resources');
+const kbDirPath = path.resolve(resourcesDirPath, 'kb');
+const documentFormatFilePath = path.resolve(kbDirPath, '_technical/toFormat/documentFormat.dcl');
+
+module.exports = {
+    extensionDevelopmentPath: extensionDevelopmentPath,
+    launchArgs: [`${kbDirPath}/`, documentFormatFilePath, '--new-window', '--disable-extensions'],
+    openInFolder: true,
+  }
```
Comment (mminot-yseop) [inline client/jest-runner-mocha.config.js:13]: 🥵 Too bad you had to go to such lengths to upgrade some stuff.

**Review by mminot-yseop — APPROVED** [review]
Assuming the test-related changes were validated by the release that was just done.

**Comment by mminot-yseop** [comment]
Fail on merge but not on branch?
image

**Comment by alacambre-yseop** [comment]
:thinking: It looks like I did not correctly used promises in the tests, making one of them fail some times.

**Pull request merged**